.github/workflows/build.yaml

```
@@ -0,0 +1,54 @@
+name: build-refdocs
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+  pull_request:
+
+jobs:
+  build:
+    name: Build reference docs
+
+    strategy:
+      matrix:
+        go-version: [1.15.x]
+        platform: [ubuntu-latest]
+
+    runs-on: ${{ matrix.platform }}
+
+    env:
+      BUILD_PATH: ${{ github.workspace }}/tmp
+
+    steps:
+
+      - name: Set up Go ${{ matrix.go-version }}
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ matrix.go-version }}
+        id: go
+
+      - name: Check out reference-docs code
+        uses: actions/checkout@v2
+
+      - name: Set release version
+        id: version_value
+        run: |
+          mkdir -p ${{env.BUILD_PATH}}
+          cd ${{env.BUILD_PATH}} && wget https://raw.githubusercontent.com/kubernetes/website/master/config.toml
+          echo 'KUBE_VERSION<<EOF' >> $GITHUB_ENV
```

should this be:
```suggestion
          echo 'KUBE_VERSION=<<EOF' >> $GITHUB_ENV
```
?

---

I based this change on these examples:
https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#multiline-strings
https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#example-6